stubs/openpyxl/openpyxl/descriptors/sequence.pyi

```diff
@@ -38,7 +38,7 @@ class Sequence(Descriptor[_ContainerT]):
 
 # `_T` is the type of the elements in the sequence.
 class UniqueSequence(Sequence[set[_T]]):
-    seq_types: tuple[type[list[_T]], type[tuple[_T, ...]], type[set[_T]]]
+    seq_types: tuple[type, ...]  # defaults to `list`, `tuple`, `set`
```

Comment:
Just commented on the issue, but I think this makes the stub worse and it's better to ignore the error.

Reply:
I also think that the stubs in this particular case are best left unchanged, since any modification would reduce accuracy. It should be sufficient to change comment in `stubtest_allowlist.txt` entry explaining why we decided not to adjust these types."

Reply:
Oh, I didn't realize the types could be overwritten. If so, using `tuple[type, ...]` does make more sense.